packages/keystone/file/utils.js

```diff
@@ -1,11 +1,12 @@
-const { createReadStream, readFileSync } = require('fs')
+const { createReadStream } = require('fs')
 const path = require('path')
 const { Duplex } = require('stream')
 
 const INVISIBLE_CHARS_REGEXP = new RegExp('[\u200B-\u200D\uFEFF]', 'g')
 
 const clearString = (cell) => {
-    if (!cell) {
+    // number 0 should be parsed as '0', not ''
+    if (!cell && typeof cell !== 'number') {
```

Comment:
Where is it used? Maybe you should check for null / undefined, instead of number?

Reply:
Right now seems like you're trying to solve specific problem...

What if `false` or other false values goes here? Will it be more `if`s?

Reply:
right now only meters import using that code, so there goes strings and numbers, so these changes should be enough

made checks for null / undefined, so 'false' as boolean can be parsed.
But it's hard right now to imagine getting something other than number or string from file